src/status_im/translations/fr.cljs

```diff
@@ -256,59 +236,133 @@
    :connect                               "Connecter"
    :address                               "Adresse"
    :password                              "Mot de passe"
-   :login                                 "Connexion"
    :sign-in-to-status                     "S'inscrire à Status"
    :sign-in                               "Inscription"
```

Comment:
Could you also take this opportunity to find a better alternative to `Inscription` ? This old translation does not sound good for current usage.

Reply:
@jeluard what do you think of `Créer un compte` ? and so we can also change `S'inscrire à Status` to `Créer un compte Status` or `Créer un compte sur Status`

Reply:
AFAIK this string is used to login, not create an account.

Reply:
@jeluard Oh I see. So it should be `Se connecter` or `Connexion` and the one above `se connecter à Status`

Reply:
Right `Se connecter` sounds good